storage/sql/migrate.go

```diff
@@ -190,4 +190,13 @@ var migrations = []migration{
 			);`,
 		},
 	},
+	{
+		stmts: []string{`
+			alter table client
+				add column saml_init_redirect_uri text not null default '';`,
+			`
+			alter table client
```

Comment:
I think these could be combined into a single statement, `ALTER TABLE client ADD COLUMN ..., ADD COLUMN`. Not sure if MySQL prohibits this?

Reply:
i'm not sure.. tbh i just followed the structure of the `refresh_token` migration above it which does three `alter`s each in a separate statement.